core/src/wsv.rs

```diff
@@ -552,7 +563,7 @@ impl WorldStateView {
     ///
     /// # Errors
     /// Fails if there is no domain
-    pub fn domain(&self, id: &<Domain as Identifiable>::Id) -> Result<&Domain, FindError> {
+    pub fn domain<'wsv>(&'wsv self, id: &DomainId) -> Result<&'wsv Domain, FindError> {
```

Comment:
Glad we got rid of the associated typing. It didn't make much sense anyway.

Reply:
yeah, that was my fault. I introduced that long time ago. Recently, I saw this in the documentation examples and figured it's unintelligible to a newcomer, just a complexity for no benefit.